docs/usage/self-hosted-configuration.md

```diff
@@ -785,8 +785,10 @@ Falls back to `renovate.json` if the name provided is not valid.
 
 ## onboardingNoDeps
 
-Set this to `true` if you want Renovate to create an onboarding PR even if no dependencies are found.
-Otherwise, Renovate skips onboarding a repository if it finds no dependencies in it.
+The default setting is `auto`.
+Renovate generates an onboarding PR when dependencies are found.
+To force an onboarding PR even without dependencies, set it to `enabled`.
+If `autodiscoverRepositories` is diabled, this setting is automatically switched to `enabled`.
```

Comment:
```suggestion
The `auto` setting is converted to `disabled` if `autodiscoverRepositories` is `true`, or converted to `enabled` if false.

In other words, the default behavior is:

- If you run Renovate on all discovered repositories then it will skip those without any dependencies detected, but
- If you run Renovate on _specific_ repositories then Renovate will onboard all such repositories even if no dependencies are found
```